site/content/en/docs/formats/kinetics.md

```diff
@@ -0,0 +1,59 @@
+---
+title: 'Kinetics'
+linkTitle: 'Kinetics'
+description: ''
+weight: 1
+---
+
+## Format specification
+
+Kinetics 400/600/700 is a video datasets for action recognition task.
+Dataset is available for downloading
+[here](https://www.deepmind.com/open-source/kinetics)
+
+Supported media type: `Video`
+
+Supported type of annotations: `Label`
+
+Supported attributes for labels: `time_start`, `time_end`
```

Comment:
Please format this paragraph as here: https://openvinotoolkit.github.io/datumaro/docs/formats/kitti/